### PR TITLE
final copy for review

### PR DESCRIPTION
**Pull request by LukeJKelley**
Add buttons for eventLst transltation
Move code dictionnary to its own module
Keep convertToMorse = convertoEnglish in same file, change name of module.js -> morseUtils.js
Add edge cases to test (ie what if user input chars that are not in dictionary?)

All changes made

### CODE REVIEW

File: modules/modules.test.js [hunk 1]
```diff
@@ -18,4 +18,10 @@ describe("Test if morse code translates to english correctly", () => {
       ".... . .---- .---- ---   .-...   -... ...-- -.--"
     );
   });
+  test("Outputs any symbol not in the code dictionary as it is typed", () => {
+    expect(convertToMorse("///")).toBe("/ / /");
+  });
+  test("Morse converts to english and joins the letters to form a word", () => {
+    expect(convertToEnglish("... --- ...")).toBe("sos");
+  });
 });
```
Comment (remihoeppe) [inline modules/modules.test.js:35]: Add Tests to make sure that sentences get translated back to English properly. Also check that special char are translated in English correctly

File: modules/morseUtils.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export const convertToMorse = (str) => {
+  return str
+    .toUpperCase()
+    .split("")
+    .map((el) => {
+      if (code[el]) {
+        return code[el];
+      }
+      return el;
```
Comment (remihoeppe) [inline modules/morseUtils.js]: Could replace this if with a lazy evaluation
`=> {
return code[el] || el;
}`

if code[el] is true it gets returned if not el is returned

File: modules/array.js [hunk 3]
```diff
@@ -53,33 +52,3 @@ export const code = {
   $: "...-..-",
   _: "..--.-",
 };
```
Comment (remihoeppe) [inline modules/array.js]: array.js is not a descriptive enough name for this file. 
Could you think more of the actual content of the array in this file rather than the data structure in it?
You will have many files with many arrays in them. So it would be best if you were more specific.
dictionary.js  would be a more appropriate name in this case

File: modules/modules.test.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { convertToMorse } from "./modules.js";
+import { convertToMorse, convertToEnglish } from "./morseUtils.js";
```
Comment (remihoeppe) [inline modules/modules.test.js:1]: This file will need to be renamed. 
Test files need to be named after the files they are testing (ie if you are testing functions contained in morseUtils.js then this needs to be named morseUtils.test.js)

**Review by remihoeppe — CHANGES_REQUESTED** [review]

File: modules/modules.test.js (hunk 1 above)
Comment (LukeJKelley) [reply modules/modules.test.js:35]: Added two new test cases, one for "hello world" and another to handle symbols, changed the function so when two spaces are entered a new word begins

File: modules/morseUtils.js (hunk 2 above)
Comment (LukeJKelley) [reply modules/morseUtils.js]: thankyou, I've done this

File: modules/array.js (hunk 3 above)
Comment (LukeJKelley) [reply modules/array.js]: I see what you're saying, new name is morse-english-dictionary.js

File: modules/modules.test.js (hunk 4 above)
Comment (LukeJKelley) [reply modules/modules.test.js:1]: Changed modules.test.js to morseUtils.js to follow best practice
